### PR TITLE
Add  signet feature

### PR DESCRIPTION
**Pull request by w-i-n-s**
Resolve https://github.com/Sjors/nthkey-ios/issues/26
- [x] Export keys to Specter
- [x] Import wallet from Specter

### CODE REVIEW

**Comment by Sjors** [comment]
This can be rebased now that #22 landed.

File: NthKey.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		2AEDBE2C25FB3F0B0018C5CF /* AddressesViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AEDBE2B25FB3F0B0018C5CF /* AddressesViewModel.swift */; };
 		2AEECEEE262C6A4100578819 /* String.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AEECEED262C6A4000578819 /* String.swift */; };
 		2AF86BCF2608B18000A770AD /* DataManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AF86BCE2608B18000A770AD /* DataManager.swift */; };
-		2AF86C34260B555C00A770AD /* Network.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AF86C33260B555B00A770AD /* Network.swift */; };
+		2AF86C34260B555C00A770AD /* NetworkOfWallet.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AF86C33260B555B00A770AD /* NetworkOfWallet.swift */; };
```
Comment (Sjors) [inline NthKey.xcodeproj/project.pbxproj:28]: Maybe `WalletNetwork`?

**Review by Sjors — COMMENTED** [review]
For the import side, I suggest duplicating the network toggle and then put it below Wallets.

When a user switches the network, it should:
1. unload the current wallet (if any)
2. load the (first) available wallet (if any)

The scan / import button can then assume the network based on the tab. When loading mainnet in testnet/signet, or the other way around, we can throw an error. For loading signet in testnet we can't, but that's fine for now. 

Schermafbeelding 2021-04-23 om 10 49 11

File: NthKey.xcodeproj/project.pbxproj (hunk 1 above)
Comment (w-i-n-s) [reply NthKey.xcodeproj/project.pbxproj]: fixed in 54c4d9d7cdfd54ce303c33ef566a40af85dd2395

**Comment by w-i-n-s** [comment]
> This can be rebased now that #22 landed. I also cherry-picked [9a5a4f3](https://github.com/Sjors/nthkey-ios/commit/9a5a4f3796e796f44359a9d7150b323de7f07920) into master. Can you split the "add signet" bit into a new commit?

@Sjors Now, is it looks okay?

**Comment by Sjors** [comment]
> Can you split the "add signet" bit into a new commit?

It's still one commit.

**Comment by w-i-n-s** [comment]
> > Can you split the "add signet" bit into a new commit?
> 
> It's still one commit.

Ah, my bad. Then I'll prepare all refactoring Network -> WalletNetwork and after can split creation a `.signet` case. Will it be okay?

**Comment by w-i-n-s** [comment]
@Sjors 
> For the import side, I suggest duplicating the network toggle and then put it below Wallets.

Just checking twice: is it the right place for it?

Снимок экрана 2021-04-23 в 17 04 50

**Comment by Sjors** [comment]
@w-i-n-s yes that's fine. But let's add a section name "Import wallet" above the tab, so it's more clearly separated from the list of wallets. And then let's move the Wallets section down, so it's right above "Wallet details".

**Comment by w-i-n-s** [comment]
> @w-i-n-s yes that's fine. But let's add a section name "Import wallet" above the tab, so it's more clearly separated from the list of wallets. And then let's move the Wallets section down, so it's right above "Wallet details".

fixed in 5af1a23e8906af1b103773f170b812e8372a109b

**Comment by w-i-n-s** [comment]
> For the import side, I suggest duplicating the network toggle and then put it below Wallets.
> 
> When a user switches the network, it should:
> 
> unload the current wallet (if any)
> load the (first) available wallet (if any)
> The scan / import button can then assume the network based on the tab. When loading mainnet in testnet/signet, or the other way around, we can throw an error. For loading signet in testnet we can't, but that's fine for now.

@Sjors I follow your idea in implementation. Please check if everything is fine in last fixes (c0718eac53f739d1800e18e4d04a7b1029357a09)

**Pull request merged**